iceberg/core/drawable.py

```diff
@@ -382,16 +382,19 @@ def pad_bottom(self, padding: float):
         """Pad the drawable on the bottom by the specified amount."""
         return self.pad((0, 0, 0, padding))
 
-    def crop(self, bounds: Bounds) -> "Drawable":
+    def crop(self, bounds: Bounds, clip: bool = False) -> "Drawable":
```

Comment:
Would be nice to document what clip does here. I think it actually makes more sense to just have a `.clip(...)` on Drawable instead of hijacking the `crop` method.